### PR TITLE
updated-ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by danger-ahead**

### CODE REVIEW

**Comment by aldebaran30701** [comment]
I believe the issue template should include a message hint in the sections of expected and actual behaviour. The hint should specify that if applicable, include the time something should execute within, and specify the actual time something took to execute. 
@danger-ahead does that sound like an improvement you could add to the PR ?

**Comment by danger-ahead** [comment]
> I believe the issue template should include a message hint in the sections of expected and actual behaviour. The hint should specify that if applicable, include the time something should execute within, and specify the actual time something took to execute.
> @danger-ahead does that sound like an improvement you could add to the PR ?

@aldebaran30701 will do that

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ maintainers' discretion.
 * Have you read Constellation's Code of Conduct? By filing an issue, you are
 expected to comply with it, including treating everyone with respect:
 https://github.com/constellation-app/constellation/blob/master/CODE_OF_CONDUCT.md
-
+* The "Steps to Reproduce" should outline the steps required to reproduce the issue. The rest should filled out accordingly.
```
Comment (antares1470) [inline ISSUE_TEMPLATE.md:11]: I would possibly move this point to immediately after the first point in the Requirements section as I think this follows on from that point

**Review by antares1470 — CHANGES_REQUESTED** [review]
To extend @aldebaran30701's comment, I'd suggest a comment including that the Expected and Actual behaviour should be measureable (which is more applicable for more issues than just time-related ones)

**Comment by antares1470** [comment]
Closing as this has been done in #1139

**Pull request closed without merging**